static/app/views/performance/data.tsx

```diff
@@ -781,8 +793,8 @@ export function generatePerformanceEventView(
 }
 
 export function generatePerformanceVitalDetailView(
-  _organization: Organization,
-  location: Location
+  location: Location,
+  isMetricsData: boolean
```

Comment:
I usually try to avoid booleans like this because this "smells" like there could be multiple types in the future, in which case passing down `metricType: "metric data" | ...` can be a bit easier to extend. If you are sure that this wont change in the future tho, totally fine to leave as is :)

Reply:
I think the boolean is ok for now...but in the future, we can maybe have something like `dataType: 'metrics' | 'transaction'`